res/Sectors/M1105/Lorspane.txt

```diff
@@ -30,375 +30,374 @@
+1319                      D635211-6 Lo                           { -3 } (611+3) [2159] - -  - 815 14 Hv G0 V M9 V      
+1320                      D753762-3 Po Cy O:1519                 { -2 } (A62+5) [3526] - M  - 603 10 Hv M1 V      
+1412                      C634104-9 Lo                           { -1 } (F01+1) [1148] - M  - 125 15 Hv G4 V      
+1413                      C200413-9 Ni Va                        { -1 } (E30+4) [535D] - -  - 405  8 Hv G8 V      
+1418                      C485405-6 Ni Pa                        { -2 } (B30+0) [82A6] - -  - 513 12 Hv M3 V K9 V      
+1420                      C321512-9 He Ni Po                     { -1 } (C42+3) [4479] - -  - 714 15 Hv G4 V      
+1513 Muadem               CAB4762-9 Fl Cy O:1518 (Muadem)        {  0 } (A66-3) [9727] - M  A 702  7 Hv M5 V K8 V      
+1515                      C7A38S4-9 Fl Ph                        {  0 } (87A+2) [B878] - -  - 201  9 Hv K3 V M7 V M5 V      
+1518 Etribb               A100301-G Lo Va                        {  2 } (D21-2) [457F] - -  - 712 12 Hv F9 V M5 V      
+1519                      A1007S4-D Na Va Pi                     {  2 } (964+0) [8929] - -  - 802  6 Hv M7 V      
+1612                      C455102-6 Lo                           { -2 } (B01+1) [1146] - -  - 910 11 Hv M6 II       
+1615                      C300503-B Ni Va                        {  0 } (943+0) [557E] - M  - 203 11 Hv M2 V      
+1616                      C326861-7 Ph Pi Cy O:1617              { -1 } (871-2) [775A] - M  - 312 12 Hv M8 III       
+1617                      A5866S6-B Ag Ni Ri                     {  3 } (D57+0) [695F] - -  - 622 16 Hv M0 V M9 V      
+1620                      A635563-D Ni Cy O:1519                 {  1 } (D46+3) [361C] - -  - 425 13 Hv M9 V      
+1711                      C400102-7 Lo Va                        { -2 } (701-1) [1159] - -  - 103 13 Hv K5 V M3 V      
+2019                      B513661-A Ic Na Ni Cy O:2124           {  1 } (A55+1) [6758] - M  - 611 10 Hv F7 V      
+2516 Haspana              A7B39T2-D Fl Hi In (Haspana)           {  4 } (888-2) [6D7F] - -  A 105 13 Hv F1 V M2 V      
+2713                      C547311-6 Lo                           { -2 } (621+4) [1156] - -  - 603  8 Hv M2 V      
+3118                      C440424-A He De Ni Po                  {  0 } (832-2) [342A] - -  - 114 16 Hv M1 V      
+3120                      C200321-6 Lo Va                        { -2 } (A21+1) [1157] - M  - 801 11 Hv M4 V      
+0121                      C98A312-B Lo Wa                        {  0 } (B21-1) [733F] - -  - 410  7 Hv M8 V      
+0126                      C896513-6 Ag Ni                        { -1 } (742-2) [5468] - -  - 700  9 Hv F4 V      
+0130                      C8697S5-7 Ri                           {  0 } (B65+1) [873B] - M  - 603 11 Hv F2 V      
+0221 Phoga                A67AAU5-G Hi In Wa                     {  5 } (99A+0) [5F9F] - E  - 903  7 Hv M8 V M8 V       
```

Comment:
Per the T5 Second Survey rules this TL should be reduced to F or E. As a Pop A world I recommend TL-E, this makes a huge difference in the overall economy of the sector.

Reply:
Updated